src/js/controller.js

```diff
@@ -449,6 +449,26 @@ export const MediaUIStates = {
     mediaEvents: ['emptied', 'loadstart'],
     videoRenditionsEvents: ['addrendition', 'removerendition'],
   },
+  MEDIA_AUDIO_TRACK_UNAVAILABLE: {
+    get: function (controller) {
+      const { media } = controller;
+
+      if (!media) return AvailabilityStates.UNSUPPORTED;
+
+      if (!media.audioTracks) {
+        return AvailabilityStates.UNSUPPORTED;
+      }
```

Comment:
suggestion(non-blocking): simplify to:

```suggestion
if (!media?.audioTracks) return AvailabilityStates.UNSUPPORTED;
```